x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/index.test.tsx

```diff
@@ -5,10 +5,138 @@
+      loadingRuleIds: [],
+      loadingRulesAction: null,
+      pagination: {
+        page: 1,
+        perPage: 20,
+        total: 1,
+      },
+      rules: [
+        {
+          actions: [],
+          created_at: '2020-02-14T19:49:28.178Z',
+          created_by: 'elastic',
+          description: 'jibber jabber',
+          enabled: false,
+          false_positives: [],
+          filters: [],
+          from: 'now-660s',
+          id: 'rule-id-1',
+          immutable: true,
+          index: ['endgame-*'],
+          interval: '10m',
+          language: 'kuery',
+          max_signals: 100,
+          name: 'Credential Dumping - Detected - Elastic Endpoint',
+          output_index: '.siem-signals-default',
+          query: 'host.name:*',
+          references: [],
+          risk_score: 73,
+          rule_id: '571afc56-5ed9-465d-a2a9-045f099f6e7e',
+          severity: 'high',
+          tags: ['Elastic', 'Endpoint'],
+          threat: [],
+          throttle: null,
+          to: 'now',
+          type: 'query',
+          updated_at: '2020-02-14T19:49:28.320Z',
+          updated_by: 'elastic',
+          version: 1,
+        },
+      ],
+      selectedRuleIds: [],
+    })),
+  };
+});
+
+jest.mock('../../../../containers/detection_engine/rules', () => {
+  return {
+    useRules: jest.fn().mockReturnValue([
+      false,
+      {
+        page: 1,
+        perPage: 20,
+        total: 1,
+        data: [
+          {
+            actions: [],
+            created_at: '2020-02-14T19:49:28.178Z',
+            created_by: 'elastic',
+            description: 'jibber jabber',
+            enabled: false,
+            false_positives: [],
+            filters: [],
+            from: 'now-660s',
+            id: 'rule-id-1',
+            immutable: true,
+            index: ['endgame-*'],
+            interval: '10m',
+            language: 'kuery',
+            max_signals: 100,
+            name: 'Credential Dumping - Detected - Elastic Endpoint',
+            output_index: '.siem-signals-default',
+            query: 'host.name:*',
+            references: [],
+            risk_score: 73,
+            rule_id: '571afc56-5ed9-465d-a2a9-045f099f6e7e',
+            severity: 'high',
+            tags: ['Elastic', 'Endpoint'],
+            threat: [],
+            throttle: null,
+            to: 'now',
+            type: 'query',
+            updated_at: '2020-02-14T19:49:28.320Z',
+            updated_by: 'elastic',
+            version: 1,
+          },
+        ],
+      },
+    ]),
+    useRulesStatuses: jest.fn().mockReturnValue({
+      loading: false,
+      rulesStatuses: [
+        {
+          current_status: {
+            alert_id: 'alertId',
+            bulk_create_time_durations: ['2235.01'],
+            gap: null,
+            last_failure_at: null,
+            last_failure_message: null,
+            last_look_back_date: '2020-03-19T00:32:07.996Z',
+            last_success_at: '03/19/2020T00:32:07.996Z',
```

Comment:
These are usually ISO date times no?

```ts
new Date().toISOString()
```

You can keep it like that as I don't think we do strict validation checking just yet in some areas with runtime. Just mentioning.

Reply:
Updated to ISO date times. I re-ran the tests and saw a warning on this. Thanks!